### ISSUE TITLE
suggestion: note which libraries are Vue 3 compatible

### ISSUE THREAD

**Issue by robrich** [vuejs/awesome-vue#3345]
Imagine this scenario: I'm enamored with Vite and therefore use Vue 3 exclusively. I'm in search of a material design framework, and this list suggests Vuetify. I install Vuetify only to hit https://github.com/vuetifyjs/vuetify/issues/11346. Crash-n-burn.

Proposed solution: let authors or curators identify when libraries support Vue 3.

**Comment by Scrum** [comment]
Hi, I believe that the author should be interested in supporting his package with different versions.

**Comment by robrich** [comment]
Agreed.  But because one initiates a global component library differently, calling out whether the library supports Vue 2 or 3 or both could be really helpful..

**Comment by Scrum** [comment]
> But because one initiates a global component library differently, calling out whether the library supports Vue 2 or 3 or both could be really helpful..

Of course this is very useful information

**Comment by andy3520** [comment]
+1 
Really need this

**Comment by VishalSharma2000** [comment]
any updates on list of libraries supported with Vue 3. Facing problems with Vuetify, Quasar, etc.

**Comment by Jogai** [comment]
See https://github.com/vuejs/awesome-vue/issues/3544

**Comment by VishalSharma2000** [comment]
Okay @Jogai. Thank you.

**Comment by vdejager** [comment]
and almost 2 years later...........this is one of the major issues our students encounter when learning Vue.

**Comment by Jogai** [comment]
That will teach them something about maintainability right away, plus a few other valuable lessons.
